INSTALL

```diff
@@ -421,6 +421,10 @@ MFEM_USE_PUMI = YES/NO
    models and effectively supports automated adaptive analysis. PUMI enables
    support for parallel unstructured mesh modifications in MFEM.
 
+MFEM_USE_UMPIRE = YES/NO Enables support for Umpire, which is a resource
+   management library that allows the discovery, provision, and management of
+   memory on machines with multiple memory devices like NUMA and GPUs.
```

Comment:
The description should start on a new line, after`YES/NO`.

Reply:
`done`